classes/coc/script/Eval.as

```diff
@@ -131,10 +183,27 @@ public class Eval {
 			case 'and':
 				return x && y;
 			default:
-				throw error(src, expr, "Unregistered operator " + op, false);
+				throw new Error("Unregistered operator " + op);
 		}
 	}
-	private function evalExpr():Function {
+	private function calculate(x:Function, op:String, y:Function):* {
+//		trace("Evaluating " + (typeof x) + " " + x + " " + op + " " + (typeof y) + " " + y);
```

Comment:
Please remove this, or replace it with a Logger (See [contributing](../../../blob/master/CONTRIBUTING.md#logging))